network/server_test.go

```diff
@@ -363,40 +365,51 @@ func WaitUntilPeerConnectsTo(ctx context.Context, srv *Server, id peer.ID) (bool
 // TestPeerReconnection checks whether the node is able to reconnect with bootnodes on losing all active connections
 func TestPeerReconnection(t *testing.T) {
 	conf := func(c *Config) {
-		c.MaxPeers = 2
+		c.MaxPeers = 3
 		c.NoDiscover = false
 	}
 	//Create bootnode
 	bootNode := CreateServer(t, conf)
-
+	bootNode2 := CreateServer(t, conf)
 	conf1 := func(c *Config) {
-		c.MaxPeers = 2
+		c.MaxPeers = 3
 		c.NoDiscover = false
-		c.Chain.Bootnodes = []string{AddrInfoToString(bootNode.AddrInfo())}
+		c.Chain.Bootnodes = []string{AddrInfoToString(bootNode.AddrInfo()), AddrInfoToString(bootNode2.AddrInfo())}
 	}
 
 	srv1 := CreateServer(t, conf1)
 	srv2 := CreateServer(t, conf1)
 
-	//connect with the boot node
-	connectedCh := asyncWaitForEvent(srv1, 10*time.Second, connectedPeerHandler(bootNode.AddrInfo().ID))
-	assert.True(t, <-connectedCh)
+	//connect with the first boot node
+	connectedCh1 := asyncWaitForEvent(srv1, 10*time.Second, connectedPeerHandler(bootNode.AddrInfo().ID))
+	assert.True(t, <-connectedCh1)
+
+	//connect with the second boot node
+	connectedCh2 := asyncWaitForEvent(srv1, 10*time.Second, connectedPeerHandler(bootNode2.AddrInfo().ID))
+	assert.True(t, <-connectedCh2)
 
 	assert.NoError(t, srv1.Join(srv2.AddrInfo(), 5*time.Second))
-	//disconnect from the boot node
-	disconnectedCh1 := asyncWaitForEvent(srv1, 10*time.Second, disconnectedPeerHandler(bootNode.AddrInfo().ID))
-	srv1.Disconnect(bootNode.AddrInfo().ID, "bye")
 
-	assert.True(t, <-disconnectedCh1, "Failed to recieved peer disconnected event")
+	//disconnect from the first boot node
+	disconnectedCh1 := asyncWaitForEvent(srv1, 15*time.Second, disconnectedPeerHandler(bootNode.AddrInfo().ID))
+	srv1.Disconnect(bootNode.AddrInfo().ID, "Bye")
 
-	//disconnect from the second node
-	disconnectedCh2 := asyncWaitForEvent(srv1, 10*time.Second, disconnectedPeerHandler(srv2.AddrInfo().ID))
+	assert.True(t, <-disconnectedCh1, "Failed to recieve peer disconnected event")
```

Comment:
Typo here: `recieve` -> `receive`

The same typo is on line 403

Reply:
Corrected it. Thanks for the catch. Commit `[link](https://github.com/0xPolygon/polygon-sdk/pull/260/commits/abff6809bcbc43529eff0a1b051caa06584c7c7e)`